app/resources/v1/streets.js

```diff
@@ -162,7 +187,10 @@ exports.delete = async function (req, res) {
   async function deleteStreet (street) {
     let user
     try {
-      user = await User.findOne({ login_tokens: { $in: [req.loginToken] } })
+      user = await User.findOne({
+        where: { login_tokens: { $contains: [req.loginToken] } }
+        // where: { login_tokens: { [Op.contains]: [req.loginToken] } }
```

Comment:
Remove commented out line?

Reply:
What is the difference between using `$contains` and `[Op.contains]`? Why is it using `[Op.contains]` in line 154 (above)?